modApiServer/src/org/aion/api/server/types/TxRecptLg.java

```diff
@@ -1,26 +1,22 @@
-/*******************************************************************************
+/*
  * Copyright (c) 2017-2018 Aion foundation.
  *
- *     This file is part of the aion network project.
+ * This file is part of the aion network project.
```

Comment:
The license got formatted incorrectly. Please replace it with the second license from https://github.com/aionnetwork/aion/wiki/Aion-License-Header